packages/SwingSet/src/kernel/loadVat.js

```diff
@@ -85,6 +85,7 @@ export function makeVatLoader(stuff) {
   }
 
   const allowedDynamicOptions = [
+    'consensusMode',
```

Comment:
We certainly want to be able to control this mode for replay purposes, but I don't think it should be controllable through the `vatAdminVat` facet. A vat that is under consensus mode should not be able to acquire the power to break consensus by virtue of creating a new vat that is not. So I kind of think this shouldn't be set on a vat-per-vat basis. It should exist in `managerOptions`, but not be permitted in `allowedDynamicOptions`/`allowedStaticOptions`.

I'm not sure what to recommend exactly. All settings that could affect the consensus-visible activity of the system ought to be "sticky" (provided to `initializeKernel` and stored in the DB somehow) so that the kernel/vat's behavior will be consistent from one run to the next. But this control is obviously useful to override during a specialized replay action, where you know you're breaking that state vector for the benefit of debugging something. The verb to override it should maybe be distinct from the control used to set it initially.

(we need to reorganize/rethink the large variety of options we have, and the places that can provide them.. adding an option requires touching far too many files, among other problems)

Reply:
> We certainly want to be able to control this mode for replay purposes, but I don't think it should be controllable through the `vatAdminVat` facet.

Ah, I overlooked that.  Thanks for pointing it out.

> But this control is obviously useful to override during a specialized replay action, where you know you're breaking that state vector for the benefit of debugging something.

I think that is a clear description of what the `$DEBUG` environment variable is for.